bc_obps/registration/api/operator.py

```diff
@@ -19,53 +19,35 @@
 
 @router.get(
     "/operators",
-    response={200: OperatorOut, codes_4xx: Message, codes_5xx: Message},
-    url_name="get_operator_by_legal_name_or_cra",
+    response={200: Union[List[OperatorSearchOut], OperatorSearchOut], codes_4xx: Message, codes_5xx: Message},
+    url_name="get_operators_by_cra_number_or_legal_name",
```

Comment:
It is a good practice to avoid hardcoding URL names in Django. Instead, we can use the `reverse` function to dynamically generate endpoints along with their arguments. This way, if we ever need to change the name of an endpoint, we won't have to manually update its references throughout our codebase. You can find more information on this topic in the Django-ninja documentation at https://django-ninja.dev/reference/operations-parameters/?h=url_name#url_name. However, I recommend reading the more comprehensive documentation at https://docs.djangoproject.com/en/5.0/topics/http/urls/#reverse-resolution-of-urls.